OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -1,6 +1,24 @@
 # Oracle WebCenter Portal 12.2.1.4.0 on Docker
+
+## Mount a host directory as a data volume
+ The default location of the volume in container is under /var/lib/docker/volumes. There is an option to mount a directory from the host into a container as volume. In this project we will use that option for the data volume.
```

Comment:
Remove the first space and add an empty line between the heading and the paragraph. You also need to add backtick quotes around `/var/lib/docker/volumes`. Note that bind mounts (what you're referred to here) are not actually considered volumes.